src/components/lablup-loading-spinner.ts

```diff
@@ -102,6 +134,10 @@ export default class LablupLoadingSpinner extends LitElement {
       this.spinner.style.display = 'block';
     }
   }
+
+  async is_active() {
```

Comment:
Is there any reason to put `async` prefix in here?

Reply:
This code is removed.